vendor/erusev/parsedown-extra/ParsedownExtra.php

```diff
@@ -302,6 +302,10 @@ protected function inlineFootnoteMarker($Excerpt)
     protected function inlineLink($Excerpt)
     {
         $Link = parent::inlineLink($Excerpt);
+        
+        if ( is_null($Link) || is_null($Link['extent'])) {
+            return null;
+        }
```

Comment:
I think we probably should just update ParsedownExtra at this point.

Reply:
ya, I've had to do it before, it works.

Reply:
or you mean PR in that repo....if it's still active, would definitely make it easier.

Reply:
last merge I see was Oct 2021. might be wishful thinking

Reply:
looks like this issue was already fixed in there though.

Reply:
then we can run composer update. might solve #3 as well.

Reply:
totally forgot to do this